go-controller/pkg/ovn/kubevirt.go

```diff
@@ -0,0 +1,88 @@
+package ovn
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	kapi "k8s.io/api/core/v1"
+
+	kvv1 "kubevirt.io/api/core/v1"
```

Comment:
could we make this more explicit ? I don't think I'll be able to associate `kvv1` to kubevirtv1. 

I'd go straight for `kubevirtv1` .